microsoft-edge/extensions-chromium/publish/api/using-addons-api.md

```diff
@@ -21,6 +21,8 @@ To submit suggestions and feedback, enter an [Issue about the Add-ons API](https
 | Language | Package | Source code | Notes |
 | --- | --- | --- | --- |
 | Node.js | [Edge Webstore Upload](https://www.npmjs.com/package/@plasmo-corp/ewu) | [GitHub](https://github.com/plasmo-corp/edge-webstore-upload)| This is a third party library. Microsoft makes no warranties, express or implied, with respect to the information provided here. |
```

Comment:
```suggestion
| Node.js | [Edge Webstore Upload](https://www.npmjs.com/package/@plasmo-corp/ewu) | [GitHub](https://github.com/plasmo-corp/edge-webstore-upload)| This is a third-party library. Microsoft makes no warranties, express or implied, with respect to the information provided here. |
```

Reply:
@SantoshPisini - i can push commits for these PR comments if you don't get to it.  
I'll have a team member review this PR to see if they ok adding the new link/ table row.

Reply:
the format of both rows seems poor: doesn't show the name of the packages.  i will try clearer more explicit format in table. 
![image](https://github.com/MicrosoftDocs/edge-developer/assets/45407486/4e8fdb59-ec6e-4528-9a1a-2f3b68ac03f1)